src/TotkRstbGenerator/Commands/RstbCommands.cs

```diff
@@ -12,9 +12,27 @@ public class RstbCommands
         """;
 
     [Command("generate", Aliases = ["gen", "calculate", "calc"], Description = "Calculate the RSTB values for a target RomFS folder")]
-    public async Task Generate([Argument] string romfs, [Option("output", ['o'], Description = GEN_OUTPUT_DESC)] string? output, [Option("padding", ['p'], Description = "Adds padding to every generated RSTB value")] uint padding = 0)
+    public async Task Generate([Argument] string romfs, [Option("rstbs", ['r'], Description = "An list of rstb files to use to patch.")] string[]? rstbs, [Option("output", ['o'], Description = "Output Folder")] string? output, [Option("padding", ['p'], Description = "Adds padding to every generated RSTB value")] uint padding = 0)
```

Comment:
This functionality should be put in a separate command (e.g. `patch`)

Reply:
done